cfme/tests/automate/test_git_import.py

```diff
@@ -78,3 +78,69 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain, br
     view.modal.apply.click()
     assert view.provisioning_entry_point.value == ("/{}/Service/Provisioning/StateMachines/"
         "ServiceProvision_Template/CatalogItemInitialization".format(imported_domain.name))
+
+
+@pytest.mark.tier(2)
+@pytest.mark.parametrize(
+    ("url", "param_type", "param_value", "verify_ssl"),
+    [
+        (
+            'https://github.com/ramrexx/CloudForms_Essentials.git',
+            'branch',
+            'origin/cf4.1',
+            True
+        ),
+        (
+            'https://github.com/RedHatQE/ManageIQ-automate-git.git',
+            'tag',
+            '0.1',
+            False
+        ),
+        (
+            "https://github.com/RedHatQE/ManageIQ-automate-git.git",
+            "branch",
+            "origin/master",
+            False,
+        ),
+        (
+            "https://github.com/ganeshhubale/ManageIQ-automate-git.git",
+            "branch",
+            "origin/test",
+            False,
+        ),
+    ],
+    ids=["with_branch", "with_tag", "with_top_level_domain", "without_top_level_domain"],
+)
+def test_domain_import_git(
```

Comment:
Don't you need to enable `git_owner` server role here?

Reply:
Already enabled globally.